derotation/load_data/custom_data_loaders.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+from scipy.io import loadmat
+
+
+def read_randomized_stim_table(path_to_randperm):
```

Comment:
I am not sure how much care I should give to these very methods because they are specific to the way a single postdoc in a single lab saves the data. The idea is that people should re-write these methods for their own setup.
I think typing and tests should be there to verify if the new implementation other people will write works. Ideally they will be for an abstract interface of data_loaders (yet to be built).

Reply:
I understand that this implementation is specific to one person's data, but I think it's still useful to have at least some minimal docstrings, because they would be of great help when trying to adapt these functions for another project (at least they would help me understand what kind of loader functions do I need to implement in the first place).